client/src/pages/private/EOIView.js

```diff
@@ -56,19 +56,10 @@ export default () => {
   }
 
   useEffect(() => {
-    const locationRoles = {
-      region_interior: 'Interior',
-      region_fraser: 'Fraser',
-      region_vancouver_coastal: 'Vancouver Coastal',
-      region_vancouver_island: 'Vancouver Island',
-      region_northern: 'Northern',
-      none: 'None'
-    };
-
     setHealthAuthorities(
       (roles.includes("superuser") || roles.includes("ministry_of_health"))
-      ? Object.values(locationRoles) 
-      : roles.map((loc) => locationRoles[loc]).filter(Boolean)
+        ? Object.values(regionLabelsMap).filter(value => value !== 'None')
```

Comment:
I think this is incorrect, if I'm reading it right - MoH and superuser should have access to None for EEOIs

Reply:
Note: does not apply to participants, None is only needed for EEOIs